### DISCUSSION TITLE
How often are achievements updated?

### DISCUSSION THREAD

**Discussion by ale-lit** [Schweinepriester/github-profile-achievements#8] [Q&A] [answered]
Hi! How often are achievements updated in profile?

Yesterday I did a merge of branches, but a new achievement (YOLO) has not yet appeared.

**Comment by RussellDash332** [comment]
It should be fast enough. Maybe you didn't merge based on the criteria, see #6 for more information

**Comment by ale-lit** [comment]
I see that in this experiment (https://github.com/Schweinepriester/github-profile-achievements/discussions/9), the achievement was given almost instantly. Turns out I did something wrong.

**Comment by ale-lit** [comment, marked as answer]
If all conditions are met correctly, the reward appears instantly
